src/Umbraco.Web.UI.Client/src/less/components/umb-folder-grid.less

```diff
@@ -37,7 +37,9 @@
    align-items: center;
 }
 
-.umb-folder-grid__folder-icon {
+.umb-folder-grid__folder-icon,
+.umb-folder-grid__folder-icon[class^="icon-"],
+.umb-folder-grid__folder-icon[class*=" icon-"] {
```

Comment:
Just ensure it also works if the developer choose a custom folder icon (which not is in core)